stdlibs/__init__.py

```diff
@@ -6,315 +6,52 @@
-        "curses",
-        "_io",
-        "glob",
-        "pprint",
-        "_lzma",
-        "concurrent",
-        "re",
-        "venv",
-        "winreg",
-        "pyclbr",
-        "_codecs_cn",
-        "locale",
-        "_codecs_hk",
-        "resource",
-        "_codecs_kr",
-        "rlcompleter",
-        "smtplib",
-        "_strptime",
-        "_codecs_iso2022",
-        "mailcap",
-        "nntplib",
-        "statistics",
-        "_markupbase",
-        "posixpath",
-        "_compat_pickle",
-        "turtle",
-        "wsgiref",
-        "_signal",
-        "shelve",
-        "keyword",
-        "urllib",
-        "decimal",
-        "_aix_support",
-        "_ast",
-        "getopt",
-        "logging",
-        "pstats",
-        "_zoneinfo",
-        "genericpath",
-        "encodings",
-        "fcntl",
-        "_json",
-        "os",
-        "cmath",
-        "_functools",
-        "doctest",
-        "bdb",
-        "getpass",
-        "turtledemo",
-        "_sha512",
-        "trace",
-        "_uuid",
-        "posix",
-        "_codecs",
-        "_curses",
-        "html",
-        "ntpath",
-        "ensurepip",
-        "io",
-        "pathlib",
-        "stringprep",
-        "warnings",
-        "_string",
-        "__future__",
-        "contextlib",
-        "copyreg",
-        "code",
-        "csv",
-        "ast",
-        "inspect",
-        "selectors",
-        "_weakrefset",
-        "sys",
-        "configparser",
-        "gc",
-        "_curses_panel",
-        "asyncore",
-        "pydoc_data",
-        "fileinput",
-        "compileall",
-        "sched",
-        "pdb",
-        "pyexpat",
-        "unittest",
-        "sre_compile",
-        "numbers",
-        "_pydecimal",
-        "pkgutil",
-        "_tracemalloc",
-        "imghdr",
-        "nis",
-        "mailbox",
-        "graphlib",
-        "_opcode",
-        "_elementtree",
-        "_py_abc",
-        "json",
-        "cgi",
-        "marshal",
-        "socketserver",
-        "_sre",
-        "typing",
-        "xdrlib",
-        "itertools",
-        "tabnanny",
-        "spwd",
-        "_abc",
-        "opcode",
-        "_codecs_jp",
-        "_collections",
-        "distutils",
-        "_bisect",
-        "mmap",
-        "ssl",
-        "_thread",
-    }
-)
+ALL = "all"
+
+KNOWN_VERSIONS = [
+    "2.3",
+    "2.4",
+    "2.5",
+    "2.6",
+    "2.7",
+    "3.0",
+    "3.1",
+    "3.2",
+    "3.3",
+    "3.4",
+    "3.5",
+    "3.6",
+    "3.7",
+    "3.8",
+    "3.9",
+    "3.10",
+]
+
+
+def stdlib_top_level(version: Optional[str] = None):
+    if version is None:
+        version = "%d%d" % sys.version_info[:2]
+        modname = f"stdlibs.py{version}"
+    elif version == ALL:
+        modname = "stdlibs.py"
+    else:
+        version = "".join(version.split(".")[:2])
+        modname = f"stdlibs.py{version}"
+
+    __import__(modname)
+    return sys.modules[modname].top_level_names  # type: ignore
```

Comment:
Why not `return importlib.import_module(modname)`?

Reply:
When did we get that newfangled thing.  I don't want to use methods out of stdlib until they've been in there a full decade.  That one's only been there 9 years.